docs/user_guide/preprocessing/roundabout_simplification.ipynb

```diff
@@ -379,7 +373,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "It's worth investigating why some roundabouts are still missing from simplification. Since the selection of roundabouts is based on the resulting polygons after polygonizing the road network let's use the [Circular Compactness](https://docs.momepy.org/en/stable/generated/momepy.CircularCompactness.html) parameter of the resulting polygons as the main selection attribute."
+    "It's worth investigating why some roundabouts are still missing from simplification. Since the selection of roundabouts is based on the resulting polygons after polygonizing the road network let's use the circular compactness (measurable as `momepy.CircularCompactness`) parameter of the resulting polygons as the main selection attribute."
```

Comment:
We have different versions of the docs, we cannot hard-code the link to one of those.